framework/tests/all-systems/Issue953.java

```diff
@@ -1,7 +1,6 @@
 // Test case for issue 953
 // https://github.com/typetools/checker-framework/issues/953
-// @skip-test
-
+// @skip-test Crash will be fixed once both pull request #978 and #977 are merged.
```

Comment:
I'll merge https://github.com/typetools/checker-framework/pull/977 to master then pull master into this pull request.  Then remove this.